blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpRollupsQueryHandlerIntegrationTest.java

```diff
@@ -49,42 +55,43 @@ public void testSingleplotQuery() throws Exception {
         JsonParser jsonParser = new JsonParser();
         JsonObject responseObject = jsonParser.parse(responseContent).getAsJsonObject();
 
-        String expectedResponse = "{\n" +
-                "  \"unit\": \"unknown\",\n" +
-                "  \"values\": [\n" +
-                "    {\n" +
-                "      \"numPoints\": 1,\n" +
-                "      \"timestamp\": 1389211230,\n" +
-                "      \"latest\": 397\n" +
-                "    }\n" +
-                "  ],\n" +
-                "  \"metadata\": {\n" +
-                "    \"limit\": null,\n" +
-                "    \"next_href\": null,\n" +
-                "    \"count\": 1,\n" +
-                "    \"marker\": null\n" +
-                "  }\n" +
-                "}";
-        JsonObject expectedObject = jsonParser.parse(expectedResponse).getAsJsonObject();
-
-        Assert.assertEquals(expectedObject, responseObject);
-        EntityUtils.consume(query_response.getEntity());
```

Comment:
We are reading the body, so its not required.  You only need to call ```EntityUtils.consume``` if you haven't read the response body.